common/src/main/java/com/microsoft/identity/common/internal/commands/parameters/DeviceCodeFlowCommandParameters.java

```diff
@@ -0,0 +1,33 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.common.internal.commands.parameters;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+
+@Getter
+@EqualsAndHashCode(callSuper = true)
+@SuperBuilder(toBuilder = true)
+public class DeviceCodeFlowCommandParameters extends TokenCommandParameters {
```

Comment:
Do we need this class? Doesn't seem to have anything

Reply:
I decided to do it this way for two reasons:

1. `getAuthorizationRequest()` in `BaseController` takes in a `TokenCommandParameters` object as parameter, so I can't create a command parameters class for DCF that only extends `CommandParameters` and add the needed fields.

2. I looked through the code and it seemed like there weren't any commands that received a `TokenCommandParameters` object from PublicClientApplication, so I figured it would make more sense to create a new class that specifically maps to DCF. It also allows future developers to add DCF-specific parameters here if they want to implement any extra functionality.